docs/contributing.rst

```diff
@@ -326,3 +326,73 @@ escaping and without iterated replacements. An area can be delimited by ``<#name
 by as many concatenations of its contents as there were sets of variables supplied to the template system,
 each time replacing any ``<inner>`` items by their respective value. Top-level variables can also be used
 inside such areas.
+
+.. _documentation-style:
+
+Documentation Style Guide
+=========================
+
+The following are style recommendations specifically for documentation
+contributions to Solidity.
+
+British English
+---------------
+
+Use British English, unless using project or brand names. For a good guide to
+what this means read `this guide from Oxford dictionaries <https://en.oxforddictionaries.com/spelling/british-and-spelling>`_
+or `this tutorial from the British Council <https://learnenglish.britishcouncil.org/en/intermediate-grammar/british-english-and-american-english>`_.
+
+Title Case for Headings
+-----------------------
+
+Use `title case <http://titlecase.com>`_ for headings. This means capitalise all principal words in
+titles, but not articles, conjunctions, and prepositions unless they start the
+title.
+
+For example, the following are all correct:
+
+* Title Case for Headings
+* For Headings Use Title Case
+* Local and State Variable Names
+* Order of Layout
+
+Expand Contractions
+-------------------
+
+Use expanded contractions for words, for example:
+
+* "Do not" instead of "Don't".
+* "Can not" instead of "Can't".
+
+Active and Passive Voice
+------------------------
+
+Active voice is typically recommended for tutorial style documentation as it
+helps the reader understand who or what is performing a task. However, as the
+Solidity documentation is a mixture of tutorials and reference content, passive
+voice is sometimes more applicable.
+
+As a summary:
+
+* Use passive voice when describing how an aspect of Solidity or the Ethereum Virtual Machine works.
+* Use active voice when describing recommendations on how to apply an aspect of Solidity.
+
+For example, the below is in passive voice as it discusses an aspect of Solidity:
+
+  Functions can be declared ``pure`` in which case they promise not to read
+  from or modify the state.
+
+For example, the below is in active voice as it discusses an application of Solidity:
+
+  When invoking the compiler, you can specify how to discover the first element
+  of a path, and also path prefix remappings.
+
+Common Terms
+------------
+
+* Function parameters and return types, not input and output parameters.
+
+Code Examples
+-------------
+
+* Ensure that all code examples begin with the correct ``pragma`` version, which is ``pragma solidity >=0.4.0 <0.6.0;``.
```

Comment:
This actually depends on whether the code is valid in those versions or not. Perhaps: Use the pragma with the largest span in which the code is valid.